### ISSUE TITLE
app-forensics/spiderfoot-3.3: <module> import cherrypy_cors ModuleNotFoundError: No module named 'cherrypy_cors' 

### ISSUE THREAD

**Issue by wofwofwof** [pentoo/pentoo-overlay#739]
When starting spiderfoot-3.3 I get 

`~% sf                                                                                                                                                                                                                           (12:08 04.09.21)(1)
Traceback (most recent call last):
  File "/usr/share/spiderfoot/sf.py", line 27, in <module>
    import cherrypy_cors
ModuleNotFoundError: No module named 'cherrypy_cors'
`
It seems cherrypy_cors is missing.

Thank you very much for your great work.

**Comment by blshkv** [comment]
thanks, I just pushed a fix, please check

**Comment by wofwofwof** [comment]
Hello blshkv,
many thanks, the error is gone now, but a new one arises:

Traceback (most recent call last):
  File "/usr/share/spiderfoot/sf.py", line 24, in <module>
    from sflib import SpiderFoot
  File "/usr/share/spiderfoot/sflib.py", line 36, in <module>
    from networkx import nx
ImportError: cannot import name 'nx' from 'networkx' (/usr/lib/python3.9/site-packages/networkx/__init__.py)

For me that looks pretty strange, because if I do ls -l on /usr/lib/python3.9/site-packages/networkx/ I get:
total 176
34595202 drwxr-xr-x  12 root root  4096 Sep  3 22:11 ./
33267716 drwxr-xr-x 477 root root 32768 Sep  4 21:33 ../
34848922 drwxr-xr-x  21 root root  4096 Sep  3 22:11 algorithms/
34660355 drwxr-xr-x   4 root root  4096 Sep  3 22:11 classes/
33039082 -rw-r--r--   1 root root  9443 Jul  8 14:57 conftest.py
33039083 -rw-r--r--   1 root root 48304 May 31 22:28 convert_matrix.py
33039078 -rw-r--r--   1 root root 15844 Jun 22 00:08 convert.py
34955636 drwxr-xr-x   4 root root  4096 Sep  3 22:11 drawing/
33039080 -rw-r--r--   1 root root  3537 May 31 22:28 exception.py
34660467 drwxr-xr-x   4 root root  4096 Sep  3 22:11 generators/
33039079 -rw-r--r--   1 root root  2906 Jul 27 04:11 __init__.py
34832752 drwxr-xr-x   4 root root  4096 Sep  3 22:11 linalg/
34660461 drwxr-xr-x   2 root root  4096 Sep  3 22:11 __pycache__/
34840961 drwxr-xr-x   5 root root  4096 Sep  3 22:11 readwrite/
33039081 -rw-r--r--   1 root root 10006 May 31 22:28 relabel.py
34832769 drwxr-xr-x   4 root root  4096 Sep  3 22:11 testing/
34840948 drwxr-xr-x   3 root root  4096 Sep  3 22:11 tests/
34840928 drwxr-xr-x   4 root root  4096 Sep  3 22:11 utils/

```
% eix networkx 
[I] dev-python/networkx
     Available versions:  2.6.2^t {examples extras test xml yaml PYTHON_TARGETS="python3_8 python3_9 python3_10"}
     Installed versions:  2.6.2^t(10:10:53 PM 09/03/2021)(-examples -extras -test -xml -yaml PYTHON_TARGETS="python3_9 -python3_8 -python3_10")
```

Thank you very much for your help.

**Comment by blshkv** [comment]
Looks like somebody reported to upstream:
https://github.com/pentoo/pentoo-overlay/issues/739

**Comment by blshkv** [comment]
So here is the summary of the problem. Version 3.3 is calling ```from networkx import nx``` which was removed from the current networkx-2.6.2. sipderfoot fixed that problem in 3.4 but broke userhome support, see:
https://github.com/smicallef/spiderfoot/issues/1176

So I dont feel like fixing any bugs for now because upstream is doing restructuring. We will come back to it after they are done. As for now, you are on your own ;-)
